<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigDialog2</class>
 <widget class="QDialog" name="ConfigDialog2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>231</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>440</width>
    <height>231</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>440</width>
    <height>231</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Service Options</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>195</y>
     <width>231</width>
     <height>30</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>15</y>
     <width>411</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>135</y>
      <width>98</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Database path 2:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browseButton_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>132</y>
      <width>26</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browseButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>92</y>
      <width>26</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dbpathEdit">
    <property name="geometry">
     <rect>
      <x>133</x>
      <y>95</y>
      <width>237</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>95</y>
      <width>98</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Database path 1:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>55</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Max Connections:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="srvportSpin">
    <property name="geometry">
     <rect>
      <x>133</x>
      <y>14</y>
      <width>262</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>65535</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="dbpathEdit_2">
    <property name="geometry">
     <rect>
      <x>133</x>
      <y>135</y>
      <width>237</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="srvconnectSpin">
    <property name="geometry">
     <rect>
      <x>133</x>
      <y>55</y>
      <width>262</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>65535</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>13</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Server Port :</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
